test/evergreen.yml

```diff
@@ -5541,6 +5541,8 @@ buildvariants:
   tasks:
     - name: compile
     - name: make-check-test
+      # Using medium ditro as more memory is required. See WT-12378.
```

Comment:
Typo "distro" . Also maybe just keep one occurrence of this comment here, something like:

"# Using medium distro for a few tests here as more memory is required. See WT-12378."

Reply:
fixed here: [a6f2964](https://github.com/wiredtiger/wiredtiger/pull/10260/commits/a6f2964685674fa985ae51fae6ee8a8dcee8c102)